common/src/main/resources/assets/entity_model_features/lang/zh_tw.json

```
@@ -1,46 +1,249 @@
+  "entity_model_features.config.function_explanation.log": "log(x)\n取得 x 的自然對數",
+  "entity_model_features.config.function_explanation.max": "max(x, y, ...)\n從數字清單中取得最大值",
+  "entity_model_features.config.function_explanation.min": "min(x, y ,...)\n從數字清單中取得最小值",
+  "entity_model_features.config.function_explanation.pow": "pow(x, y)\n取得 x 的 y 次方",
+  "entity_model_features.config.function_explanation.print": "print(id, n, x)\n每 N 個影格列印一次值「x」",
+  "entity_model_features.config.function_explanation.printb": "printb(id, n, x)\n每 N 個影格列印一次布林值「x」",
+  "entity_model_features.config.function_explanation.quadbezier": "quadbezier(k, x, y, z)\n使用二次貝茲曲線樣條函數和控制點 z 在 x 和 y 之間插值",
+  "entity_model_features.config.function_explanation.raddiff": "raddiff(x, y)\n傳回兩個弧度值 x 和 y 之間的最短弧度差",
+  "entity_model_features.config.function_explanation.random": "random(seed)\n0 到 1 之間的隨機數。提供種子將始終傳回相同的結果。種子是選填的",
+  "entity_model_features.config.function_explanation.round": "round(x)\n將 x 捨入到最接近的整數",
+  "entity_model_features.config.function_explanation.signum": "signum(x)\n取得 x 的符號（正或負）",
+  "entity_model_features.config.function_explanation.sin": "sin(x)\n取得 x 的正弦",
+  "entity_model_features.config.function_explanation.between": "between(x, min, max)\n檢查值是否介於最小值和最大值之間",
+  "entity_model_features.config.function_explanation.equals": "equals(x, y, epsilon)\n比較兩個浮點值，帶有誤差範圍",
+  "entity_model_features.config.function_explanation.in": "in(x, val1, val2, ...)\n檢查值是否等於多個值中的一個",
+  "entity_model_features.config.function_explanation.sqrt": "sqrt(x)\n取得 x 的平方根",
+  "entity_model_features.config.function_explanation.tan": "tan(x)\n取得 x 的正切",
+  "entity_model_features.config.function_explanation.ifb": "if(cond, val, [cond2, val2, ...], val_else)\n根據一個或多個條件選擇一個值\n僅傳回布林值",
+  "entity_model_features.config.function_explanation.randomb": "random(seed)\n隨機布林值 true|false。提供種子將始終傳回相同的結果。種子是選填的",
+  "entity_model_features.config.function_explanation.catch": "catch(x,c,id)\n將會傳回 x，但是如果 x 是 NaN 或導致其他動畫錯誤，則會改為傳回 c。\nid 是選填的，如果新增它，catch 函數將會使用此 ID 將使用 c 的原因列印到遊戲記錄檔中\n主要用於動畫除錯，如果有可能觸發此情況，您應該真正修復它。",
+  "entity_model_features.config.function_explanation.todeg": "todeg(rad)\n將弧度轉換為角度",
+  "entity_model_features.config.function_explanation.torad": "torad(deg)\n將角度轉換為弧度",
+  "entity_model_features.config.function_explanation.wrapdeg": "wrapdeg(x)\n將角度值 x 限制在 -180 到 180 的範圍內",
+  "entity_model_features.config.function_explanation.wraprad": "wraprad(x)\n將弧度值 x 限制在 -pi 到 pi 的範圍內",
+  "entity_model_features.config.general": "一般",
+  "entity_model_features.config.general.tooltip": "一般設定",
+  "entity_model_features.config.large_mob_lod": "保留較大型生物的 LOD",
+  "entity_model_features.config.large_mob_lod.tooltip": "如果啟用，這將根據生物的大小\n減少上述設定的細節層級 (LOD) 影響。\n較大型生物的跳幀現象\n將比小型生物更不明顯",
+  "entity_model_features.config.lod": "動畫 LOD 距離",
+  "entity_model_features.config.lod.tooltip": "設定動畫開始跳幀的距離，\n降低細節層級 (LOD)，這有助於提升效能\n無 = 不跳幀\n16 = 每距離玩家 16 格就會跳過 1 個影格\n較小的數字表示跳過更多影格和獲得更多效能提升\n但當然動畫的影格率會越低，\nLOD 距離設定得越近",
```

我不太確定 skip frame 的中文術語什麼，先譯為「略過影格」
```suggestion
  "entity_model_features.config.lod.tooltip": "設定動畫開始略過影格的距離，\n降低細節層級 (LOD)，這有助於提升效能\n無 = 不略過影格\n16 = 每距離玩家 16 格就會跳過 1 個影格\n較小的數字表示略過更多影格和提升\n但當然動畫的影格率會越低，\nLOD 距離設定得越近",
```